bolt/include/bolt/Core/DebugData.h

```diff
@@ -197,6 +197,9 @@ class DebugRangesSectionWriter {
   /// to .debug_ranges
   uint64_t getEmptyRangesOffset() const { return EmptyRangesOffset; }
 
+  /// Returns the RangesBuffer's size.
+  uint64_t getRangesBufferSize() const { return RangesBuffer->size(); }
```

Comment:
Hmm, since you are updating SectionOffset in appendToRangeBuffer, probably just using getSectionOffset will work also? Instead of adding this new API.